media/docs/cpp/blackwell_cluster_launch_control.md

```diff
@@ -6,15 +6,15 @@ A GEMM workload usually consists of three phases: prologue, mainloop and epilogu
 
 Consider a GEMM that has `20x20x1` output tiles, running on a GPU with `100` SMs. There is another kernel occupying all the resources of `20` SMs so only `80` SMs can be used. Assume cluster shape is `1x1x1`. The following diagram shows how the schedule would look like for such a kernel. 
 
-<p align="center"><img src=../../images/non_persistent.png alt="A beautiful sunset" title="Sunset over the mountains"></p>
```

Comment:
lol. I am curious to know how this text ended up in here? 

BTW, Google search on "A beautiful sunset" GitHub gives me CUTLASS as the top result. :)

![Screenshot 2025-04-11 at 10 59 08 AM](https://github.com/user-attachments/assets/03bccc75-1ffb-4a2a-8792-803bafce15cb)

Reply:
CUTLASS dev team has impeccable taste in satire